packages/edit-widgets/src/components/edit-widgets-initializer/index.js

```diff
@@ -3,7 +3,26 @@
  */
 import Layout from '../layout';
 
+/**
+ * WordPress dependencies
+ */
+import { useDispatch } from '@wordpress/data';
+import { useEffect, useState } from '@wordpress/element';
+
 function EditWidgetsInitializer( { settings } ) {
+	// Since this module relies on the idea of using a "stub post", a state must be initialized
+	// first, or the data layer would attempt to retrieve the stub post from the API.
+	const { initializeState } = useDispatch( 'core/edit-widgets' );
+	const [ stateInitialized, setStateInitialized ] = useState( false );
+	useEffect( () => {
+		initializeState();
+		setStateInitialized( true );
+	}, [] );
+
+	if ( ! stateInitialized ) {
+		return <div></div>;
```

Comment:
Could this be `return null;`? Having an empty div in the DOM structure isn't ideal.